posthog/taxonomy/property_definition_api.py

```diff
@@ -456,15 +455,15 @@ def update(self, property_definition: PropertyDefinition, validated_data: dict):
         }
         # free users can update property type but no other properties on property definitions
         if set(changed_fields) == {"property_type"}:
-            changed_fields["updated_by"] = self.context["request"].user
             if changed_fields["property_type"] == "Numeric":
                 changed_fields["is_numerical"] = True
             else:
                 changed_fields["is_numerical"] = False
 
             return super().update(property_definition, changed_fields)
         else:
-            raise EnterpriseFeatureException()
+            # Delegate to TaggedItemSerializerMixin.update() which calls ModelSerializer.update()
```

Comment:
```suggestion
            # uses TaggedItemSerializerMixin to ensure property updates are only allowed when appropriate features are licensed for this customer/organisation
```

IMHO comments should always say why not what :)